### PR TITLE
Remove createSandbox utility

### PR DESCRIPTION
**Pull request by jhowardmsft**
Signed-off-by: John Howard <jhoward@microsoft.com>

With the dynamic sandbox management functionality, we won't need the `createSandbox` utility any more. This handles that change. I'm not where the change is for  removing prebuildSandbox.vhdx itself though.

### CODE REVIEW

**Comment by soccerGB** [comment]
I am wondering what the performance delta between the existing createSandbox utility and the replacement mechanism( and the details about this replacement)

**Comment by jhowardmsft** [comment]
> I am wondering what the performance delta between the existing createSandbox utility and the replacement mechanism( and the details about this replacement)

It will be zero - docker uses a cached copy.

**Comment by jhowardmsft** [comment]
The in-progress branch (still needs HCS functionality) is in https://github.com/Microsoft/docker/tree/jjh/sandbox

**Comment by jhowardmsft** [comment]
https://github.com/moby/moby/pull/34170 is open in docker. Once this is merged, this PR can be rebased and merged.

**Comment by jhowardmsft** [comment]
Rebased. This should be good to go in now as the dynamic sandbox management PR is in moby/moby master

File: docs/customosbuildinstructions.md [hunk 1]
```diff
@@ -93,10 +89,8 @@ Here are the expected contents of each subdirectory /file
             /bin/vhd2tar
             /bin/tar2vhd
             /bin/exportSandbox
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: Small documentation thing: can you add `/bin/remotefs` and `/bin/netnscfg` here?

File: docs/customosbuildinstructions.md [hunk 2]
```diff
@@ -93,10 +89,8 @@ Here are the expected contents of each subdirectory /file
...
-            /bin/createSandbox
-            /bin/netnscfg
 
-            Note : exportSandbox, createSandbox, vhd2tar, tar2vhd, and netnscfg are actually hard links to the file gcstools
+            Note : exportSandbox, vhd2tar and tar2vhd are actually hard links to the file gcstools
```
Comment (gupta-ak) [inline docs/customosbuildinstructions.md]: same add `remotefs` and `netnscfg`

Comment (jhowardmsft) [inline docs/customosbuildinstructions.md]: I would, but @soccerGB just merged it.....

**Pull request merged**